packages/editor/src/components/rich-text/index.native.js

```diff
@@ -179,7 +178,7 @@ export class RichText extends Component {
 			} ) );
 
 		// Save back to HTML from React tree
-		const html = '<' + tagName + '>' + toHTMLString( value ) + '</' + tagName + '>';
+		const html = '<' + tagName + '>' + value + '</' + tagName + '>';
```

Comment:
@hypest can you help testing if this still works with mobile app?

Reply:
Tested against the current mobile app master (https://github.com/wordpress-mobile/gutenberg-mobile/commit/6f5e3507ff40eb351ef7c5e4c16a9db1cdae2a07) and works fine 👍

Reply:
❤️ to both of you.